src/components/item/publish/CCLicenseSelection.tsx

```diff
@@ -119,13 +260,17 @@ const CCLicenseSelection: FC<Props> = ({ item, disabled }) => {
           <Typography variant="subtitle1">
             {translateBuilder(BUILDER.ITEM_SETTINGS_CC_LICENSE_PREVIEW_TITLE)}
           </Typography>
-          <CCLicenseIcon
-            adaption={settings?.ccLicenseAdaption as CCLicenseAdaption}
-            sx={{ mt: 1 }}
-          />
+          <Box display="flex" justifyContent="left">
+            <CreativeCommons
+              requireAccreditation={requireAttributionValue === 'yes'}
+              allowSharedAdaptation={allowSharingValue as CCSharingVariant}
```

Comment:
Can you control the type here?

Reply:
I don't think so because `allowSharingValue` can also be empty when no option is selected and it doesn't typecheck to `CCSharingVariant`.